package.json

```diff
@@ -40,7 +37,7 @@
     "blake2b": "^2.1.4",
     "decimal.js": "^10.4.3",
     "dotenv": "^16.4.4",
-    "ethers": "^5.7.2",
+    "ethers": "^6.11.1",
```

Comment:
This is a breaking change and I recall that it really changes the API I hope you did this for good reason!

Reply:
`5.x` seems to be marked legacy so eventually we should bump it up but right now that breaks the code, as it is in the latest commit of that branch.